packages/app/src/app/index.js

```diff
@@ -133,7 +133,8 @@ function boot() {
 
     try {
       const { signals, store } = controller.provide();
-      render(
+      const root = createSyncRoot(rootEl);
+      root.render(
```

Comment:
You forgot to remove `rootEl` as the second param to `root.render()` 😀

Reply:
Ohhh boy!

Reply:
I reverted this because the function does not exist, I think react dom needs to be upgraded

Reply:
That makes sense. This change (render to sync root) is really important though, so maybe we can bump to the 16.9 rc?

Reply:
Hmm, I did bump the versions, let me try again...